Firestore/core/src/model/document.h

```diff
@@ -17,125 +17,57 @@
 #ifndef FIRESTORE_CORE_SRC_MODEL_DOCUMENT_H_
 #define FIRESTORE_CORE_SRC_MODEL_DOCUMENT_H_
 
-#include <iosfwd>
-#include <memory>
+#include <iostream>
```

Comment:
`<iostream>` is a relatively heavy-weight header, so if `<iosfwd>` suffices, it's better to use `<iosfwd>`. It looks like you can do that if you move the definition of `operator<<` into the source file.

Reply:
I did not address this yet as this would require me to create `document.cc` and I am not sure if that tradeoff is worth making.

Reply:
I don't think there's any harm in creating a `.cc` file, even if it's just for one function.

Reply:
Done